CHANGELOG.md

```diff
@@ -5,6 +5,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [0.39.4] - 2024-12-23
+#### ✨ Highlights
+To finish up the last release's Christmas present, the environment installation for macOS just got another huge speedup! 🚀
```

Comment:
maybe we can change it to something along the lines:

`Last release got additional speedup!`